README.md

```diff
@@ -16,3 +16,37 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+I would verify this claim by using the algorithm to sort all permutations of
+lists of varying sizes. If X does run on $O(n)$, all permutations of an input
+size n will take around the same amount of time, and as the input size 
```

Comment:
Hmm, are you sure that the same input sizes would take the same amount of time? Is that true for any of the sorting algorithms we talked about?

Reply:
Have a look at the slide where I showed you some empirical results for the different sorting problems. Did the same input size really take the same time?